teps/0091-trusted-resources.md

```diff
@@ -141,25 +135,17 @@ In later iterations, this can be extended such that tag-referenced images can
 _themselves_ be verified for the overall Task to pass verification. However,
 this will be left out of the scope of this TEP.
 
-The Pipeline Author will be able to provide optional configuration to Tekton
-Chains such that Chains will *only* consider results from Tasks that were
-verifiable (and, following the previous rule, were successfully verified).
-
 
 ### Risks and Mitigations
 
-* Mutating webhook is a risk of the proposal. The proposed verification will happen after the mutating webhook, so the content of a Resource may be mutated and fail the verification. This would be an issue for local cluster resources.
+* Mutating webhook is a risk of the proposal. The proposed verification will happen after the mutating webhooks, so the content of a Resource may be mutated and fail the verification. This would be an issue for local cluster resources. And it also means that if we don't address this issue, all signed resources need to be signed again everytime we change the mutating webhook.
 
-  Some possible mitigations include the following:
+  This TEP proposes to call `SetDefaults` when signing the resources and skip the mutating webhooks when the trusted resource feature is enabled.
```

Comment:
A caveat to this is skipping the mutating webhook would technically be a backwards incompatible change since it alters the response back to clients (though I highly doubt users are relying on its behavior - AFAICT it only sets param/result types).

Fine while it's opt-in, but if we do want to make the 2 paths consistent (which I'm in favor of) we should plan to start those deprecation timers ASAP.

Reply:
We just want to skip the mutating for tasks and pipelines, for taskrun and pipelinerun we can still have the mutating webhook. So removing it seems not ok?